libraries/AP_Camera/AP_Camera_SoloGimbal.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <GCS_MAVLink/GCS_MAVLink.h>
+
+class AP_Camera_SoloGimbal {
+public:
+
+    // GoPro status reported in the GoPro heartbeat message
+    enum STATUS {
```

Comment:
These are duplicating mavlink enumerations.  We should be using those instead of duplicating them here.

Reply:
OK.  So would the status instead something along the lines if the status is `GOPRO_HEARTBEAT_STATUS.GOPRO_HEARTBEAT_STATUS_CONNECTED`?
https://mavlink.io/en/messages/ardupilotmega.html#GOPRO_HEARTBEAT_STATUS